lxd/db/cluster/entities.go

```diff
@@ -644,6 +649,18 @@ func GetEntityURLs(ctx context.Context, tx *sql.Tx, projectName string, entityTy
 	var args []any
 	result := make(map[entity.Type]map[int]*api.URL)
 
+	// If the server entity type is in the list of entity types, or if we are getting all entity types and
```

Comment:
shall we land this separately too - doesn't feel tightly coupled to the new API endpoints and tables?

Reply:
This one is a little more tricky to separate than the other. The `server` entity type is added in this PR and isn't required beforehand. Shall I do a separate PR to add `entity.TypeNetworkZone`, `entity.TypeImageAlias`, and `entity.TypeServer`?

Reply:
yes makes sense thanks